src/components/manage/productions/index/Events.js

```diff
@@ -52,6 +47,11 @@ const Event = ({
     setIsExpanded((prevValue) => !prevValue);
   };
 
+  const type = useMemo(() => {
+    const typeId = terms.find((term) => term.domain === 'eventtype')?.id ?? '';
+    return t(`offerTypes*${typeId}`, { keySeparator: '*' });
```

Comment:
This was necessary  because the id's contain '.' which i18n uses as a keySeparator

Reply:
```suggestion
    // The custom keySeparator was necessary because the ids contain '.' which i18n uses as default keySeparator
    return t(`offerTypes*${typeId}`, { keySeparator: '*' });
```

Reply:
Thanks for the explanation! Adding it to the code could help prevent confusion in the future.